uaclient/apt.py

```diff
@@ -408,12 +408,15 @@ def setup_apt_proxy(
     :param https_proxy: the url of the https proxy apt should use, or None
     :return: None
     """
+    if http_proxy or https_proxy:
+        print(status.MESSAGE_SETTING_SERVICE_PROXY.format(service="apt"))
```

Comment:
This should be capitalized "APT" for human-readable representation of the APT-service stack. I think we generally have used the appropriate service capitalization when representing human-facing prompts/info.